modules/ROOT/pages/installing-mobile-services.adoc

```diff
@@ -10,16 +10,139 @@ NOTE: {org-name} recommends running OpenShift using the method described here. H
 
 * MacOS or Linux
 
-* Docker
+* link:https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html[Ansible] version 2.3 or later
 
-. Install OpenShift using the instructions at https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md
+* link:https://www.openshift.org/download.html[Openshift client tools] version 3.9 or later
 
+* Docker must be version 17.09.1 in order to be compatible with the latest release of OpenShift and depending on your OS this is downloaded from different places:
 
+[role="primary"]
+.MacOS
 
-== Next Steps
+****
+You can download Docker CE for MacOS version 17.09.1 link:https://download.docker.com/mac/stable/19543/Docker.dmg[here].
+****
+
+[role="secondary"]
+.Linux
+
+****
+You can download Docker CE for Linux version 17.09.1 link:https://download.docker.com/linux/centos/7/x86_64/stable/Packages/docker-ce-17.09.1.ce-1.el7.centos.x86_64.rpm[here].
```

Comment:
This link is only for CentOS and perhaps Fedora and RHEL.
I feel like we should note it down and say for other distros, go to Docker installation guide: https://docs.docker.com/install/